jerry-core/lit/lit-magic-strings.c

```diff
@@ -114,6 +83,16 @@ lit_get_magic_string_utf8 (lit_magic_string_id_t id) /**< magic string id */
 lit_utf8_size_t
 lit_get_magic_string_size (lit_magic_string_id_t id) /**< magic string id */
 {
+  static const lit_magic_size_t lit_magic_string_sizes[] =
+  {
+#define LIT_MAGIC_STRING_DEF(id, utf8_string) \
+    sizeof(utf8_string) - 1,
+#include "lit-magic-strings.inc.h"
+#undef LIT_MAGIC_STRING_DEF
+  };
+
+  JERRY_ASSERT (id < LIT_MAGIC_STRING__COUNT);
+
```

Comment:
The removed `lit_magic_strings_init` initialized the `ecma_magic_string_max_length` global variable (in debug build; and the var is only declared in debug builds). However, this replacement leave that variable uninitialized. I guess that it would not be that simple to calculate the max length at compile time, but then the var should be removed and all the code that depends on it.